katello/katello-service

```diff
@@ -14,6 +14,7 @@ SERVICES = {
   'pulp_workers'          => 20,
   'pulp_celerybeat'       => 20,
   'pulp_resource_manager' => 20,
+  'postgresql'            => 20,
```

Comment:
I don't *think* there's anything before this point that needs postgresl today, but I think it might be better if we do the databases early to avoid such a problem later.  What do you think about including it at the same level as mongod?